kartothek/io/dask/bag.py

```diff
@@ -55,6 +55,7 @@ def read_dataset_as_metapartitions_bag(
     predicates=None,
     factory=None,
     dispatch_by=None,
+    partition_size=None,
 ):
     """
     Retrieve dataset as `dask.bag` of `MetaPartition` objects.
```

Comment:
that's more a nitpick from the last PR but should be fixed here anyway:
both function actually need the following section in the docstrings to get the parameter list rendered correctly:

```
Parameters
----------
```

(yes, the empty section is enough)

Also, the return type description is missing entirely.

Reply:
Thanks for pointing that out, I will include a "Parameters" and "Return Type" section. 
Anyway, did I get it right that the _actual_ documentation of the parameters should be done with the `default_docs` decorator?

Reply:
(Meaning, I do not have to copy and paste the whole description of parameters here?)

Reply:
Yes, `@default_docs` takes care of the parameter descriptions . You can verify the docstring is how you want it to be by using the ipython `?` magic e.g.`read_dataset_as_metapartitions_bag?`, this way you don't need to build the docs